### PR TITLE
Add Makefile for developers

### PR DESCRIPTION
**Pull request by cob16**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -9,15 +9,9 @@ jobs:
...
-      - run:
-          name: Spin up docker instances
-          command: docker-compose up -d ; sleep 20
       - run:
           name: Run tests
-          command: docker run --env UH_CONNECTION_STRING="Data Source=tcp:stubuniversalhousing;Initial Catalog=StubUH;Integrated Security=False;User ID=sa;Password=Rooty-Tooty" --network repo_default --name lbhtenancyapitest --interactive --rm lbhtenancyapitest dotnet test
-      - run:
-          name: Lint
-          command: echo "Please ensure your project has no warnings generated by the code style analysis - config files are included in the project."
+          command: docker-compose run -e UH_CONNECTION_STRING="Data Source=tcp:stubuniversalhousing;Initial Catalog=StubUH;Integrated Security=False;User ID=sa;Password=Rooty-Tooty" --rm lbhtenancyapitest dotnet test
```
[the remark this answers is not in the document]
Comment (cob16) [reply .circleci/config.yml]: that's what the links are for in the docker compose

[the remark this answers is not in the document]
Comment (StevenLeighton21) [reply .circleci/config.yml]: I missed that, solid upgrade 👍

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+.PHONY: test
+test:
+	docker-compose run -e UH_CONNECTION_STRING=$(TEST_UH_CONNECTION_STRING) --rm lbhtenancyapitest dotnet test
+
+.PHONY: shell
+shell:
+	docker-compose run --rm lbhtenancyapi /bin/bash
+
+.PHONY: check
+check: lint test
```
Comment (StevenLeighton21) [inline Makefile]: Will this work or look for a `lint` command?

Comment (cob16) [reply Makefile]: Gonna ask Jeff if we can get linting in the build

Comment (StevenLeighton21) [reply Makefile]: We explored it a lot early on in the project, turns out cli driven linters aren't really a thing and the packages we tried aren't really what we were looking for. Linting is generally done inside Visual Studio and lint checks being part of deployment process didn't seem common in C# projects. We did some config to tweak the default rules to suit us and left it to an 'honour system' of linting, as anyone opening the project in VS or Rider should also have the same rule set.

**Pull request merged**